packages/docs/stories/components/radio-group.stories.ts

```diff
@@ -251,13 +276,23 @@ export const CustomValidity: Story = {
   </form>`,
 };
 
+export const Checked: Story = {
+  render: () => html`
+  <syn-radio-group label="This is a label" help-text="This is checked" name="a" value="2">
+    <syn-radio value="1">Option</syn-radio>
+    <syn-radio value="2">Option</syn-radio>
+    <syn-radio value="3">Option</syn-radio>
+  </syn-radio-group>`,
+};
+
 // Bundled screenshot story
 /* eslint-disable sort-keys */
 export const Screenshot: Story = generateScreenshotStory({
   Default,
   Labels,
   HelpText,
   Disabled,
+  Sizes,
   Checked,
 }, 230);
```

Comment:
The height value for the screenshot of the stories need to be adapted. The last stories are displayed in each other. see the last stories of https://64f819d70a69cb8728e06daf-yynihmhahw.chromatic.com/?path=/story/components-syn-radio-group--screenshot

<img width="253" height="467" alt="Image" src="https://github.com/user-attachments/assets/cc177ca7-bc60-4e16-b26f-10d3a62b6915" />

A value of `400` would be sufficient.

```suggestion
}, 400);
```

Reply:
Fixed